packages/pagination/examples/basic.md

```diff
@@ -93,3 +95,112 @@ initialState = {
   </Row>
 </Grid>;
 ```
+
+### Cursor Pagination
+
+```jsx
+const { Well } = require('@zendeskgarden/react-notifications/src');
+const { Toggle, Field, Label, Range } = require('@zendeskgarden/react-forms/src');
+const { getColor } = require('@zendeskgarden/react-theming/src');
+
+const StyledCircle = styled.div`
+  margin: 0 3px;
+  border-radius: 100%;
+  height: ${props => props.theme.space.base * 3}px;
+  width: ${props => props.theme.space.base * 3}px;
+`;
+
+const StyledCircleStroke = styled(StyledCircle)`
+  border: ${props =>
+    `0.6px ${props.theme.borderStyles.solid} ${getColor('grey', 300, props.theme)}`};
+`;
+
+const StyledCircleFill = styled(StyledCircle)`
+  background: ${props => getColor('blue', 600, props.theme)};
+`;
+
+const StyledFlexContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  margin: ${props => `${props.theme.space.base * 5}px 0`};
+`;
+
+const VisuallyHidden = styled.div`
+  border: 0;
+  clip: rect(0 0 0 0);
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  width: 1px;
+  white-space: nowrap;
+  word-wrap: normal;
+`;
+
+const CursorPaginationExample = () => {
+  const [cursor, setCursor] = React.useState(0);
+  const [startLast, setStartLast] = React.useState(true);
+  const circles = [0, 1, 2, 3, 4];
+  const onNext = () => {
+    if (cursor < circles.length - 1) {
+      setCursor(cursor + 1);
+    }
+  };
+  const onPrevious = () => {
+    if (cursor > 0) {
+      setCursor(cursor - 1);
+    }
+  };
+  const onFirst = () => setCursor(0);
+  const onLast = () => setCursor(circles.length - 1);
+
+  return (
+    <Grid>
+      <Row alignItems="center">
+        <Col size="auto" className="u-mb u-mt">
+          <Well isRecessed style={{ width: 200 }}>
+            <Field>
+              <Toggle checked={startLast} onChange={event => setStartLast(!startLast)}>
+                <Label>Show First & Last</Label>
+              </Toggle>
+            </Field>
+          </Well>
+        </Col>
+        <Col alignSelf="center" className="u-mb u-mt">
+          <StyledFlexContainer>
+            {circles.map((circle, index) =>
+              index === cursor ? (
+                <StyledCircleFill key={index} />
+              ) : (
+                <StyledCircleStroke key={index} />
+              )
+            )}
+          </StyledFlexContainer>
+          <VisuallyHidden aria-live="polite">{`The cursor is ${cursor}`}</VisuallyHidden>
```

Comment:
This visually hidden element provides audio affordance for users who visit the example page on assistive devices.